cronos/assetlist.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_id": "cronosmainnet_25-1",
+    "assets": [
+      {
+        "description": "Cronos Chain - Get Instant DApp Portability with EVM Support",
+        "denom_units": [
+          {
+            "denom": "basecro",
```

Comment:
So the CRO on Cronos is distinct? It's not IBCed over CRO from Crypto.org chain?

Are they convertable? If so, how?

Reply:
Hello Sunny, the token from `Crypto.org Chain -> Cronos` is indeed IBC token and there is a auto conversion done by `x/cronos` that takes care of the denom and the decimal (see this [link](https://github.com/crypto-org-chain/cronos/blob/main/x/cronos/spec/01_concepts.md)), i.e. It will still be `basecro` over there on Cronos 👍

Reply:
Got it, thanks!  This is really helpful (and cool!)

I guess next step is we should figure out how to add CRC20 tokens to the assetlists, but we don't have to block this PR on that